Cargo.toml

```diff
@@ -41,7 +41,7 @@ rpath = false
 blake2 = { git = "https://github.com/near/near-blake2.git", version = "0.9.1", default-features = false }
 borsh = { version = "0.8.2", default-features = false }
 bn = { package = "aurora-bn", git = "https://github.com/aurora-is-near/aurora-bn.git", default-features = false }
-evm = { git = "https://github.com/aurora-is-near/sputnikvm", rev = "2a8a3e9", default-features = false }
+evm = { version = "0.26.0", default-features = false }
```

Comment:
I think we should use 0.26.1?

Reply:
<img width="615" alt="Screenshot 2021-05-12 at 12 37 10" src="https://user-images.githubusercontent.com/28708963/117961772-c75bb900-b31e-11eb-8415-04d04d6e79ed.png">

Reply:
No, thats for `evm-core` not `evm`.

Reply:
https://github.com/aurora-is-near/aurora-engine/commit/a51008afb54e0ff467dc1ed8592116891a47f4b2#r50714346